cmd/api/src/database/assetgrouptags.go

```diff
@@ -358,6 +359,16 @@ func (s *BloodhoundDB) GetAssetGroupTagSelectorsByTagId(ctx context.Context, ass
 	return results, nil
 }
 
+func (s *BloodhoundDB) GetSelectorsByMemberId(ctx context.Context, memberId int, assetGroupTagId int) (model.AssetGroupTagSelectors, error) {
+	var selectors model.AssetGroupTagSelectors
+
+	return selectors, CheckError(s.db.WithContext(ctx).Raw(`
+		SELECT s.* from asset_group_tag_selectors s JOIN 
+		asset_group_tag_selector_nodes n ON s.id = n.node_id JOIN 
+		asset_group_tags t ON s.asset_group_tag_id = t.id WHERE 
+		t.id = ? AND n.node_id = ?`, assetGroupTagId, memberId).Find(&selectors))
+}
```

Comment:
Stylistically what do you think of this? There's several good ways to format queries tho!
```suggestion
	return selectors, CheckError(s.db.WithContext(ctx).Raw(`
		SELECT s.* from asset_group_tag_selectors s
		JOIN asset_group_tag_selector_nodes n ON s.id = n.selector_id
		JOIN asset_group_tags t ON s.asset_group_tag_id = t.id
		WHERE t.id = ? AND n.node_id = ?
		`, assetGroupTagId, memberId).Find(&selectors))
}
```

Reply:
that definitely looks prettier 😅